scripts/pre_commit_linter.py

```diff
@@ -821,8 +821,14 @@ def _check_docstrings(all_files):
     summary_messages = []
     files_to_check = [
         filename for filename in all_files if not
-        any(fnmatch.fnmatch(filename, pattern) for pattern in EXCLUDED_PATHS)
+        (any(
+            fnmatch.fnmatch(filename, pattern) for pattern in EXCLUDED_PATHS))
+        and not (filename.endswith('_check_docs_utils.py'))
         and filename.endswith('.py')]
+    # The file _check_docs_utils.py has to be excluded from this check
```

Comment:
But now this file is under our control, right? So we can do it according to our code style.

Reply:
No, we should not make a habit of disabling checks like this (in general). There are ways around this, e.g. you could write the multiline string a different way:

```
    [
        '...',
        '...'
    ].join('\n')
```

But, at worst, you would disable just the multiline docstring check for those two lines. It does not warrant disabling lint checks for the entire file.

Reply:
Fixed, used `'''` instead if `"""`. Done!